tests/integration-tests/conftest.py

```diff
@@ -363,24 +368,44 @@ def _config_renderer(config_file="pcluster.config.ini", **kwargs):
         env = Environment(loader=file_loader)
         rendered_template = env.get_template(config_file).render(**{**kwargs, **default_values})
         config_file_path.write_text(rendered_template)
-        add_custom_packages_configs(config_file_path, request, region)
-        _enable_sanity_check_if_unset(config_file_path)
```

Comment:
you removed this but you are not enabling sanity check anywhere else. Shall we do it when we call the CLI commands that accept it?

Reply:
you are right, just disregard my comment.